libafl/src/feedbacks/mod.rs

```diff
@@ -23,6 +23,8 @@ pub use new_hash_feedback::NewHashFeedbackMetadata;
 
 #[cfg(feature = "nautilus")]
 pub mod nautilus;
+#[cfg(feature = "std")]
+pub mod stdio;
```

Comment:
Should I add `pub use stdio::{StdOutToMetadataFeedback, StdErrToMetadataFeedback};` here?

Reply:
Don't think it'll be used _that_ often, right? But up to you

Reply:
Yeah, will leave it as is then.